public/css/main.css

```diff
@@ -0,0 +1,123 @@
+/* Project: Personal Portfolio
+   Author: Calvince
+   Date: 6/02/2021
+*/
+
+/* Imports */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+/* General body rules */
+
+body {
+  background: #e5e5e5;
+}
+
+/* Main container rules */
+
+.container {
+  position: relative;
+  width: 100vw;
+  height: 100vh;
+  background: #fff;
+  min-width: 375px;
+}
+
+/* Top Navigation Rules */
+#toolbar {
+  min-height: 48px;
+  margin: 0 auto;
+  position: relative;
+  top: 44px;
+}
+
+.menu-items {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  color: #28352f;
+  padding: 10px 18px;
+}
+
+.logo {
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.hamburger {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  height: 40px;
+  width: 40px;
+  cursor: pointer;
+}
+
+.hamburger > span {
+  position: absolute;
+  border-radius: 2px;
+  border: 2px solid;
+  width: 100%;
+  left: 4px;
+}
+
+.hamburger span:nth-child(1) {
+  top: 0;
+}
+
+.hamburger span:nth-child(2) {
+  top: 8px;
+}
+
+.hamburger span:nth-child(3) {
+  top: 16px;
+}
+
+/* Headline section */
+
+.headline {
+  position: relative;
+  top: 84px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 800;
+  font-size: 48px;
+  line-height: 58px;
+  letter-spacing: 0.35px;
+  color: #172b4d;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 10px 18px;
+}
+
+.headline span {
+  color: #36b37e;
+}
+
+.headline p {
+  font-size: 17px;
+  color: #172b4d;
+  line-height: 30px;
+  margin: 40px 0;
+}
+
+/* Social Media Icons */
+
+.social {
+  width: 200px;
+  min-height: 25px;
+  position: relative;
+  top: 100px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+}
+
+.social span {
+  color: #505f79;
+  font-size: 22px;
+  font-weight: 900;
+}
```

Comment:
- [ ] Positioning elements with `position` is good, but also we should make use of `margin` and it's variants to accommodate our elements in the page's layout.
- [ ] Remember to use `background-image` to add the missing background image to the `about me` section

Reply:
@jubaan 
I would like to point out that the margin is also used

Reply:
I have noticed, `margin` is sufficiently used in different classes. I think this issue is resolved now! :handshake: